ods_ci/tasks/Resources/Provisioning/Hive/AWS/provision-gpu.sh

```diff
@@ -21,17 +21,18 @@ oc get -o json -n openshift-machine-api $SOURCE_MACHINESET  | jq -r > /tmp/sourc
 OLD_MACHINESET_NAME=$(jq '.metadata.name' -r /tmp/source-machineset.json )
 NEW_MACHINESET_NAME=${OLD_MACHINESET_NAME/worker/gpu}
 
+
 # Change instanceType and delete some stuff
-jq -r ".spec.template.spec.providerSpec.value.instanceType = \"$INSTANCE_TYPE\"
+jq -r --arg INSTANCE_TYPE "$INSTANCE_TYPE" '.spec.template.spec.providerSpec.value.instanceType=$INSTANCE_TYPE
   | del(.metadata.selfLink)
   | del(.metadata.uid)
   | del(.metadata.creationTimestamp)
   | del(.metadata.resourceVersion)
-  " /tmp/source-machineset.json > /tmp/gpu-machineset.json
+  | .spec.template.spec.taints += [{"effect": "NoSchedule" , "key": "nvidia.com/gpu" , "value": "None"}]
```

Comment:
don't we need also the `operator` attribute? 

I guess `operator: "Exists"` (without `"value": "None"`) or `operator: "Equal"` (with `"value": "None"`)?

Reply:
nope..this is same as what get assigned for managed and i wanted to keep it consistent

Reply:
> as what get assigned for managed 


what do you mean?

Reply:
I mean the taint which has been applied here is same as managed cluster which is `nvidia.com/gpu=NONE:NoSchedule` . I wanted to keep the consistency